src/base1/cockpit.css

```diff
@@ -276,10 +276,13 @@ a {
 
 /* Alert fixups */
 
+/* FIXME: word-wrap workaround for Patternfly issue 491 */
```

Comment:
I suggest using "HACK" here, since that is what we usually write and using the actual [issue link](https://github.com/patternfly/patternfly/issues/491) for convenience:
`/* HACK: word-wrap workaround for long alerts https://github.com/patternfly/patternfly/issues/491`